test/js-api/wasm-module-builder.js

```diff
@@ -262,6 +268,11 @@ class WasmModuleBuilder {
     return this.setTableBounds(length);
   }
 
+  // TODO(ssauleau): legacy, remove this
```

Comment:
Can we drop this?

Reply:
Sure, the comment or the method?

Reply:
the method, if possible.

Reply:
Ok, the method is actually not used in this repo but is in wpt (I don't know if v8 does). I think that it would be simpler to later synchronize if we keep this method. Files would stay similar. What do you think?

Reply:
Not sure what exactly you are proposing. V8 does use the method, but that's fine since it tests v8-specific messages. spec or wpt should not use it, and if it's still defined in wpt we should aim to remove it there instead of adding it here (in spec).

Reply:
Right, I agree. I'll remove it here and followup on the wpt side to remove it.